ChangeLog

```diff
@@ -49,6 +49,7 @@
 	* resume data no longer has timestamps of files
 	* require C++11 to build libtorrent
 
+	* fix crash caused by empty bitfield 
```

Comment:
Wait, then I didn't understand the changelog. I guess changes on top fixes on bottom.
could you discribe what is on top and what on bottom?